wpilibj/src/main/java/edu/wpi/first/wpilibj/motorcontrol/PWMMotorController.java

```diff
@@ -73,6 +74,15 @@ public double get() {
     return m_pwm.getSpeed() * (m_isInverted ? -1.0 : 1.0);
   }
 
+  /**
+   * Common interface for getting the current voltage of the motor controller.
+   *
+   * @return The current controller voltage output. Value is between -12V and 12V.
+   */
+  double getVoltage() {
```

Comment:
This should be a public method, not package-private.
```suggestion
  public double getVoltage() {
```